packages/core/js-client/src/api.spec.ts

```diff
@@ -65,8 +65,16 @@ describe("User API methods", () => {
 
       const typedServices: Record<string, ServiceDef> = services;
 
+      if (!("demoCalc" in functions)) {
+        throw new Error("Aqua code must contain demoCalc function");
+      }
+
       const { script } = functions["demoCalc"];
 
+      if (!("Calc" in typedServices)) {
+        throw new Error("Aqua code must contain Calc service");
+      }
+
```

Comment:
both can be asserts, because it's inside test. You can leave it as it currently is as well

Reply:
Would prefer to do assert in test